src/plugins/timeConductor/DateTimePopupFixed.vue

```diff
@@ -0,0 +1,368 @@
+
+      const bound =
+        this.timeSystemFormatter.parse(this.formattedBounds[`${startOrEnd}Date`]) +
+        this.timeSystemDurationFormatter.parse(this.formattedBounds[`${startOrEnd}Time`]);
+      const timeStampString = this.timeSystemFormatter.format(bound);
+
+      try {
+        await navigator.clipboard.writeText(timeStampString);
+      } catch (err) {
+        this.openmct.notifications.error('Failed to copy timestamp to clipboard');
+        console.error(err);
+      }
+    },
+    async pasteFromClipboard(startOrEnd) {
+      if (startOrEnd !== 'start' && startOrEnd !== 'end') {
+        console.warn('Invalid startOrEnd value');
+        return;
+      }
+
+      let timeStampString;
+      try {
+        timeStampString = await navigator.clipboard.readText();
+      } catch (err) {
+        this.openmct.notifications.error('Failed to get timestamp from clipboard');
+        console.error(err);
+        return;
+      }
+
+      if (!this.timeSystemFormatter.validate(timeStampString)) {
+        this.openmct.notifications.warn(`"${timeStampString}" is not a valid timestamp format`);
+        return;
+      }
+
+      const bound = this.timeSystemFormatter.parse(timeStampString);
+      this.formattedBounds[`${startOrEnd}Date`] = this.timeSystemFormatter.formatDate(bound);
+      this.formattedBounds[`${startOrEnd}Time`] = this.timeSystemDurationFormatter.format(
+        Math.abs(bound),
+        'HH:mm:ss.SSS'
+      );
+
+      this.validateInput([`${startOrEnd}Date`, `${startOrEnd}Time`]);
+      this.reportValidity([`${startOrEnd}Date`, `${startOrEnd}Time`]);
+    },
+    setViewFromBounds() {
+      this.formattedBounds = {
+        startDate: this.timeSystemFormatter.formatDate(this.bounds.start),
+        startTime: this.timeSystemDurationFormatter.format(Math.abs(this.bounds.start)),
+        endDate: this.timeSystemFormatter.formatDate(this.bounds.end),
+        endTime: this.timeSystemDurationFormatter.format(Math.abs(this.bounds.end))
+      };
+    },
+    setBoundsFromView(shouldDismiss) {
+      if (this.$refs.fixedDeltaInput.checkValidity()) {
+        const start =
+          this.timeSystemFormatter.parse(this.formattedBounds.startDate) +
+          this.timeSystemDurationFormatter.parse(this.formattedBounds.startTime);
+        const end =
+          this.timeSystemFormatter.parse(this.formattedBounds.endDate) +
+          this.timeSystemDurationFormatter.parse(this.formattedBounds.endTime);
+
+        const bounds = { start, end };
+        this.timeContext.setBounds(bounds);
+      }
+
+      if (shouldDismiss) {
+        this.$emit('dismiss');
+
+        return false;
+      }
+    },
+    clearAllValidation() {
+      Object.keys(this.inputValidityMap).forEach(this.clearValidation);
+    },
+    clearValidation(refName) {
+      const input = this.getInput(refName);
+
+      input.setCustomValidity('');
+      input.title = '';
+    },
+    submitForm(shouldDismiss) {
+      this.validateLimit();
```

Comment:
`this.validateLimit` is expecting a `bounds` parameter. Is this intentionally dropped here?

Reply:
I've fixed `validateLimit` and `validateBounds`. Passed to the component the proper requirements to validate limit. Fixed an issue where going back and forth between bounds and limit failures wasn't notifying properly.